.cypress/integration/VisualizationCharts/10_horizontalBar_chart.spec.js

```diff
@@ -0,0 +1,202 @@
+  beforeEach(() => {
+    renderHorizontalBarChart();
+  });
+
+  it('Render horizontal bar chart and verify by default the data gets render', () => {
+    cy.get('.xy').should('exist');
+  });
+
+  it('Render horizontal bar chart and verify you see data configuration panel and chart panel', () => {
+    cy.get('.euiPanel.euiPanel--paddingSmall').should('have.length', numberOfWindow);
+    cy.get('.euiTitle.euiTitle--xxsmall').contains('Configuration').should('exist');
+    cy.get('.euiTitle.euiTitle--xxsmall').contains('dimensions').should('exist');
+    cy.get('.euiTitle.euiTitle--xxsmall').contains('breakdowns').should('exist');
+    cy.get('.euiButton__text').contains('Update chart').should('exist');
+
+    cy.get('.euiIEFlexWrapFix').contains('Panel options').click();
+    cy.get('.euiIEFlexWrapFix').contains('Legend').click();
+    cy.get('.euiIEFlexWrapFix').contains('Chart styles').click();
+    cy.get('.euiIEFlexWrapFix').contains('Color theme').click();
+  });
+
+  it('Render horizontal bar chart and verify the data configuration panel and chart panel are collapsable', () => {
+    cy.get('.euiPanel.euiPanel--paddingSmall').should('have.length', numberOfWindow);
+    cy.get('[aria-label="Press to toggle this panel"]').eq(0).click({ force: true });
+    cy.get('[aria-label="Press to toggle this panel"]').eq(1).click({ force: true });
+  });
+});
+
+describe('Render horizontal bar chart for data configuration panel', () => {
+  beforeEach(() => {
+    renderHorizontalBarChart();
+  });
+
+  it('Render horizontal bar chart and verify data config panel', () => {
+    cy.get('[data-test-subj="explorer__vizDataConfig-panel"]').eq(0).should('contain', 'tags');
+    cy.get('[data-test-subj="explorer__vizDataConfig-panel"]').eq(0).should('contain', 'count');
+    cy.get('[data-test-subj="explorer__vizDataConfig-panel"]').eq(0).should('contain', 'avg bytes');
+    cy.get('[data-test-subj="explorer__vizDataConfig-panel"]').eq(0).should('contain', 'host');
+  });
+
+  it('Render horizontal bar chart and verify data config panel restrict user to select a duplicate field on dimension field', () => {
+    cy.get('[data-test-subj="viz-config-delete-btn"]').eq(2).click();
+    cy.get('[data-test-subj="viz-config-add-btn"]').eq(4).click();
+    cy.get('[data-test-subj="comboBoxSearchInput"]').eq(0).click();
+    cy.get('.euiComboBoxOption__content').contains('host');
+  });
+
+  it('Render horizontal bar chart and verify data config panel Restrict user to select a duplicate field on Series field', () => {
+    cy.get('[data-test-subj="viz-config-add-btn"]').eq(0).click();
+    cy.get('[data-test-subj="comboBoxSearchInput"]').eq(0).click();
+    cy.get('.euiComboBoxOption__content').contains('count');
+    cy.get('.euiComboBoxOption__content').should('not.contain', 'tags');
+  });
+
+  it('Render horizontal bar chart and verify data config panel no result found if metric is missing', () => {
+    cy.wait(delay);
```

Comment:
is this wait required?

Reply:
Removed it and seems to still run reliably.